Dungeoneer/src/com/interrupt/dungeoneer/dto/LookAtDTO.java

```diff
@@ -0,0 +1,27 @@
+package com.interrupt.dungeoneer.dto;
+
+import com.badlogic.gdx.graphics.Color;
+
+public class LookAtDTO {
+    private String title;
+    private String attributes;
+    private Color color;
+
+    public LookAtDTO(String title, String attributes, Color color) {
+        this.title = title;
```

Comment:
Since this is usually something like: `E: use` etc should we call this `prompt`?